src/main/java/no/unit/nva/model/contexttypes/Book.java

```diff
@@ -2,6 +2,7 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import no.unit.nva.model.contexttypes.utils.BookSeriesUtil;
```

Comment:
In line 103, in the copy() method, we can remove the `throws InvalidIsbnException`